luigi_pipeline/lib/model/sv_mt_schema.py

```diff
@@ -61,11 +61,12 @@ def __init__(self, *args, gene_id_mapping=None, **kwargs):
         super().__init__(*args, **kwargs)
         self._gene_id_mapping = gene_id_mapping
 
+    @row_annotation()
```

Comment:
The idea here is to make `end_locus` a real annotation and then add it to the matrix table's `key` when we export the `Variants` mt.  The `key` will persist when we create the `Genotypes` mt. 

Then, when we elastic search export (which joins the `Variants` and `Genotypes` tables), the `end_locus` field will be dropped alongside `locus` and `alleles`.

We actually do have test coverage to verify that the exported variant fields has not changed ([here's](https://github.com/broadinstitute/seqr-loading-pipelines/pull/461/files#diff-51c65deaa50a78ae1b01e0b6b370d1186ba53f59842232a1372f283eef64e689R331-R362) the list).

Reply:
Also.... is `end_locus` even the right field to add to the key?

Reply:
Exact coordinates for SVs are not precise enough to use as keys, and given that the start position isn't part of the key it seems pretty counterintuitive for the end position to be part of the key. I really think the unique string identifier that we currently use as the key should be the only field we key by